src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -117,13 +122,59 @@ export class QuestionSubmissionFormComponent implements DoCheck {
   }
 
   /**
-   * Sorts recipients of feedback by their name.
+   * Compares FeedbackResponseRecipients' names using localeCompare.
+   */
+  private compareByName(firstRecipient: FeedbackResponseRecipient,
+     secondRecipient: FeedbackResponseRecipient): number {
+    return firstRecipient.recipientName.localeCompare(secondRecipient.recipientName);
+  }
+
+  /**
+   * Compares FeedbackResponseRecipients' sections using localeCompare.
+   */
+  private compareBySection(firstRecipient: FeedbackResponseRecipient,
+     secondRecipient: FeedbackResponseRecipient): number {
+
+    if (firstRecipient.recipientSection && secondRecipient.recipientSection) {
+      return firstRecipient.recipientSection.localeCompare(secondRecipient.recipientSection);
+    }
+
+    if (firstRecipient.recipientSection) {
+      return -1;
+    }
+
+    if (secondRecipient.recipientSection) {
+      return 1;
+    }
+
+    return 0;
+  }
+
+  /**
+   * Compares FeedbackResponseRecipients' teams using localeCompare.
    */
-  private sortRecipientsByName(): void {
-    this.model.recipientList.sort((firstRecipient: FeedbackResponseRecipient,
-      secondRecipient: FeedbackResponseRecipient) =>
-      firstRecipient.recipientName.localeCompare(secondRecipient.recipientName));
+  private compareByTeam(firstRecipient: FeedbackResponseRecipient,
+     secondRecipient: FeedbackResponseRecipient): number {
+
+    if (firstRecipient.recipientTeam && secondRecipient.recipientTeam) {
+      return firstRecipient.recipientTeam.localeCompare(secondRecipient.recipientTeam);
+    }
+
+    if (firstRecipient.recipientTeam) {
+      return -1;
+    }
+
+    if (secondRecipient.recipientTeam) {
+      return 1;
+    }
 
+    return 0;
+  }
+
+  /**
+   * Updates the recipientSubmissionForms to correspond to the recipientList.
```

Comment:
Applies everywhere in this PR: in general, if the method header parrots the method name and does not value add, it's better to just not have it.

Reply:
@wkurniawan07 In the most recent commit I removed the method headers I wrote which are similar to method name. However I left the ones made by previous committers. Please let me know if the latter should are also be removed.